cmd/bundle/summary.go

```diff
@@ -31,7 +33,37 @@ func newSummaryCommand() *cobra.Command {
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		b, diags := prepareBundleForSummary(cmd, forcePull, includeLocations)
-		return renderBundle(cmd, b, diags, true)
+		var err error
+
+		if root.OutputType(cmd) == flags.OutputText {
+			err = render.RenderDiagnostics(cmd.OutOrStdout(), b, diags, render.RenderOptions{RenderSummaryTable: false})
+		} else {
+			err = render.RenderDiagnostics(cmd.OutOrStderr(), b, diags, render.RenderOptions{RenderSummaryTable: false})
```

Comment:
The call in if and else conditions are the same, should there be `RenderSummaryTable: true` somewhere?

Reply:
It's not the same: stderr for json, stdout for test.

Reply:
Oh right, it was difficult to spot